### DISCUSSION TITLE
How do you update?

### DISCUSSION THREAD

**Discussion by TheMissingPort** [UltimaHoarder/UltimaScraper#1201] [General]
All I do is redownload the code and replace my profiles in new folder. Is there an easier way? Thanks

**Comment by LunarPenguin** [comment]
You could look into using git to update the scraper

**Reply by freakish804** [reply]
sorry for my ignorance buddy. could you please lemme know what's git and how to use it to update?

**Reply by LunarPenguin** [reply]
I was kinda expecting this response tbh.

I would suggest you read the [wikipedia article](https://en.wikipedia.org/wiki/Git) about it since it explains it in way more detail then i ever could here.

i would also suggest you watch a youtube video about it.

But judging by your comment and what i expect to be the response for the majority of the users of the software i think it would be better for @DIGITALCRIMINAL to somehow integrate updating the software into the software itself although i think that may take some considerable time.
